content/hi/docs/contribute/localization.md

```diff
@@ -0,0 +1,435 @@
+
+प्रत्येक टीम के लिए, अनुरोध किए गए GitHub उपयोगकर्ताओं की सूची 
+[GitHub में अपनी स्थानीयकरण टीम जोड़ें](#add-your-localization-team-in-github),
+वर्णमाला क्रम में जोड़ें ।
+
+```diff
+--- a/OWNERS_ALIASES
++++ b/OWNERS_ALIASES
+@@ -48,6 +48,14 @@ aliases:
+     - stewart-yu
+     - xiangpengzhao
+     - zhangxiaoyu-zidif
++  sig-docs-es-owners: # Admins for Spanish content
++    - alexbrand
++    - raelga
++  sig-docs-es-reviews: # PR reviews for Spanish content
++    - alexbrand
++    - electrocucaracha
++    - glo-pena
++    - raelga
+   sig-docs-fr-owners: # Admins for French content
+     - perriea
+     - remyleone
+```
+
+### एक PR खोलें
+
+`kubernetes/website`रिपोजिटरी में स्थानीयकरण जोड़ने के लिए इसके बाद, [एक PR खोलें](/docs/contribute/new-content/open-a-pr/#open-a-pr) (PR)।
+PR को स्वीकृत करने से पहले पीआर में सभी [न्यूनतम आवश्यक कंटेंट](#minimum-required-content) शामिल होनी चाहिए।
+
+एक नया स्थानीयकरण जोड़ने के उदाहरण के लिए, सक्षम करने के लिए PR देखें
+[फ्रेंच में डॉक्स](https://github.com/kubernetes/website/pull/12548)।
+
+### एक स्थानीयकृत README फ़ाइल जोड़ें
+
+अन्य स्थानीयकरण योगदानकर्ताओं का मार्गदर्शन करने के लिए,
+[`README-**.md`](https://help.github.com/articles/about-readmes/) के शीर्ष स्तर पर एक नया
+[kubernetes/website](https://github.com/kubernetes/website/), जोड़ें जहां `**` दो अक्षरों वाला भाषा कोड है।
+उदाहरण के लिए, एक जर्मन रीडमी फ़ाइल `README-de.md` होगी।
+
+स्थानीयकृत `README-**.md` फ़ाइल में स्थानीयकरण योगदानकर्ताओं को मार्गदर्शन प्रदान करें।
+`README.md` में निहित समान जानकारी को भी शामिल करें :
+
+- स्थानीयकरण परियोजना के लिए संपर्क का एक बिंदु
+- स्थानीयकरण के लिए विशिष्ट कोई भी जानकारी
+स्थानीयकृत README बनाने के बाद, मुख्य अंग्रेजी `README.md` से फ़ाइल में एक लिंक जोड़ें,
+और अंग्रेजी में संपर्क जानकारी शामिल करें।  आप एक GitHub आईडी, ईमेल पता,
+[स्लैक चैनल](https://slack.com/), या संपर्क का कोई अन्य तरीका प्रदान कर सकते हैं।  स्थानीयकृत सामुदायिक आचार संहिता (community code of conduct) में आपको अपना लिंक भी देना होगा।
+
+### अपना नया स्थानीयकरण लॉन्च करना
+
+एक बार स्थानीयकरण  वर्कफ़्लो और न्यूनतम आउटपुट
+- वेबसाइट पर भाषा चयन सक्षम करें
+- [Cloud Native Computing Foundation](https://www.cncf.io/about/)(CNCF) चैनलों के माध्यम से स्थानीयकरण की उपलब्धता को प्रचारित करें, जिसमें 
+  [कुबेरनेट्स ब्लॉग](/blog/) शामिल है।
+
+## अनुवाद विषय
+
+कुबेरनेट्स प्रलेखन के *सभी* का स्थानीयकरण करना एक बहुत बड़ा कार्य है। छोटे से शुरू करना ठीक है और
+समय के साथ विस्तार करें।
+
+### न्यूनतम आवश्यक कंटेंट
+
+कम से कम, सभी स्थानीयकरणों में शामिल होना चाहिए:
+
+विवरण | URLs
+-----|-----
+होम| [सभी शीर्षक और उपशीर्षक URL](/docs/home/)
+स्थापित करना | [सभी शीर्षक और उपशीर्षक URL](/docs/setup/)
+ट्यूटोरियल | [कुबेरनेट्स बेसिक्स](/docs/tutorials/kubernetes-basics/), [हैलो मिनिक्यूब](/docs/tutorials/hello-minikube/)
+एक नई स्थानीयकृत TOML फ़ाइल में साइट स्ट्रिंग्स | [सभी साइट स्ट्रिंग्स](#site-strings-in-i18n) विज्ञप्ति | [सभी शीर्षक और उपशीर्षक URL](/releases) ।
```

Comment:
Where we talk specifically about English pages, we should leave their titles in English.